baseplate/sidecars/event_publisher.py

```diff
@@ -161,6 +165,33 @@ def publish(self, payload: SerializedBatch) -> None:
         raise MaxRetriesError("could not sent batch")
 
 
+def build_batch_and_publish(
```

Comment:
it might be useful to add some kind of sleep command here to avoid running the CPU at 100% the entire time

Reply:
hmm, I am nervous about changing the behavior of the sidecar as a side effect of this change - I believe it is still consistent with its behavior currently as-is